docs_src/sql_databases/sql_app/crud.py

```diff
@@ -24,6 +24,25 @@ def create_user(db: Session, user: schemas.UserCreate):
     return db_user
 
 
+def update_user(db: Session, user: schemas.UserUpdate):
+    db_user = db.query(models.User).filter(user.id == user.id).one()
+
+    # Update model class variable from requested fields
+    for var, value in vars(user).items():
+        setattr(db_user, var, value) if value else None
```

Comment:
`value and setattr(db_user, var, value)` go ahead :)

Reply:
True, thank you. I will make this change and fix some failing test this weekend.